fastestimator/op/tensorop/augmentation/cutmix_batch.py

```
@@ -14,21 +14,17 @@
 # ==============================================================================
 from typing import Any, Dict, Iterable, List, Optional, Tuple, TypeVar, Union
```

Ruff (0.11.9)

`typing.TypeVar` imported but unused

Remove unused import: `typing.TypeVar`

(F401)